### PR TITLE
Grouped transactions of same coin

### PR DESCRIPTION
**Pull request by nicolas-t**
Grouped all transactions of the same coin under one single table row.
You can click it, to expand it and see the details of all the transactions made for this coin, 
Once expanded if you click one of the transaction you can still edit or remove it.

I made a lot of helpers in `trade_view.ex` dunno if it the right place (my first time with elixir and phoenix), maybe it would have been better in a specific service.

Feel free to move the, code of course :)

### CODE REVIEW

File: assets/js/components/coin-transactions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const toggleCoinTrades = function () {
+	const currencyId = this.getAttribute('data-currency-id')
```
Comment (Secretmapper) [inline assets/js/components/coin-transactions.js]: Can we get this properly indented? (2 spaces)

**Review by Secretmapper — COMMENTED** [review]
Phenomenal work! Once you just edit the js indent levels, I'll merge this right away.

Cheers!

File: assets/js/components/coin-transactions.js (hunk 1 above)
Comment (nicolas-t) [reply assets/js/components/coin-transactions.js]: Thanks !
Updated in 24eb08b

**Comment by Secretmapper** [comment]
Thanks @nicolas-t!

**Pull request merged**